pkg/driver/kubernetes/driver.go

```diff
@@ -69,21 +70,27 @@ type Driver struct {
 	podChooser           podchooser.PodChooser
 	eventClient          clientcorev1.EventInterface
 	userSpecifiedRuntime bool
+	userSpecifiedConfig  bool
 	namespace            string
 	loadbalance          string
 	authHintMessage      string
 }
 
 func (d *Driver) Bootstrap(ctx context.Context, l progress.Logger) error {
 	return progress.Wrap("[internal] booting buildkit", l, func(sub progress.SubLogger) error {
-
 		_, err := d.configMapClient.Get(ctx, d.configMap.Name, metav1.GetOptions{})
+
 		if err != nil {
```

Comment:
It's a little unclear to me how this works. What happens if a serious error is passed back here but userSpecifiedConfig is set?

Reply:
Good point - it would probably be masked - let me refine this a tad more to validate the error specifically.